### PR TITLE
[JENKINS-67933] Move plugin manager search bar back to middle

### PR DESCRIPTION
**Pull request by timja**
See [JENKINS-67933](https://issues.jenkins.io/browse/JENKINS-67933).

image

### Proposed changelog entries

* Move plugin manager search bar back to middle (regression in 2.325).

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change) and are in the imperative mood. [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate. 
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are accurate, human-readable, and in the imperative mood
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

### CODE REVIEW

File: war/src/main/js/plugin-manager-ui.js
Comment (timja) [inline war/src/main/js/plugin-manager-ui.js:66]: all the other pages autofocused this input on load so I made that consistent at the same time. (this fix alone would have addressed 'mouse-moving' concerns :)

File: war/src/main/js/plugin-manager-ui.js
Comment (timja) [inline war/src/main/js/plugin-manager-ui.js:7]: this relied on the javascript being loaded in the page after the input was defined, moved it to only try access it when it exists so moving the input around doesn't break the JS

**Review by NotMyFault — APPROVED** [review]

**Review by basil — APPROVED** [review]
Thank you Tim, this was driving me nuts. =)

---

This PR is now ready for merge. We will merge it after approximately 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process. Thanks!

**Review by kaka1365 — APPROVED** [review]
Good

**Comment by kaka1365** [comment]
Okay

File: war/src/main/js/plugin-manager-ui.js
Comment (kaka1365) [reply war/src/main/js/plugin-manager-ui.js:7]: Okay

**Review by alecharp — APPROVED** [review]

**Pull request merged**
